generator/internal/genclient/model.go

```diff
@@ -78,6 +78,8 @@ type APIState struct {
 	MessageByID map[string]*Message
 	// EnumByID returns a message that is associated with the API.
 	EnumByID map[string]*Enum
+	// Package is the proto package name.
+	Package string
```

Comment:
With OpenAPI specifications there is no package name.  For the Rust codec I chose to return either `api.Name` or (if set) `codec.PackageNameOverride`.  I think we should do something consistent across codecs for this.

Separately: I think APIState does not carry its weight.  We may want to refactor those fields in `API`.

Reply:
removed. Was not needed after all.